src/logic/algo/kernels.ts

```diff
@@ -0,0 +1,5 @@
+export const ubcKernel = (
```

Comment:
Rozumiem że tutaj docelowo będzie ten wzorek z ubc? Będziemy zakładać stałe "c" takie jakie my wybierzemy tak?

Reply:
W sumie to też mógłby być hiperparamter
Nie pomyślałem o tym

Ale raczej na razie nie będę tego wprowadzać, bo to by wymagało trochę komplikacji (doszły by kolejne abstrakcje hiperparametrów do hiperparametrów)

Więc myślę, że na razie można zostawić tak, jak jest

Reply:
W sensie, że my będziemy ustawiać c na sztywno